app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1186,6 +1191,9 @@ class BrowserTabViewModel @Inject constructor(
                         }
                     }
                 }
+
+                // new page should always show an omnibar
+                command.value = ShowOmnibar
```

Comment:
why is this needed?

Reply:
It fixes https://app.asana.com/0/1207418217763355/1208316032398497/f.

Reply:
I asked @joshliebe to look at this too, it might affect the loading bar experiment he’s running.

Reply:
@0nko I tested this and the experiment works fine if `ChangeOmnibarPositionFeature` isn’t enabled. But when the loading bar is at the bottom there is always a delay, regardless of the whether the experiment is enabled or not.

You can test this by:

- Removing this from `LoadingBarExperimentVariantInitializer`:
```
loadingBarExperimentFeature.self().isEnabled() &&
loadingBarExperimentFeature.allocateVariants().isEnabled()
```
- And changing this line `val probabilities = doubleArrayOf(0.0, 1.0)` in `generateRandomBoolean()` so it becomes the test variant
- And finally, changing “Network download rate limit” in Android developer settings to the slowest and clicking a link.
- When the loading bar is at the top there is never a delay when a link is clicked, when it’s at the bottom there is a delay.

Reply:
@joshliebe Should be fixed now [here](https://github.com/duckduckgo/Android/pull/4885/commits/2714c263b1129262f72b4dbd9f78e2df9fad1159).

Reply:
@0nko I tested this and it’s working fine for the test variant but it’s also showing immediately for the control variant now (Which still needs the delay like before). This is happening if it’s at the bottom or top.